src/components/IconComponent.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react'
+import { styled } from '@mui/material'
+import MaterialIcon from './MaterialIcon'
+import { Color } from '../types/mui-color'
+
+const StyledErrorIcon = styled(
+  ({
+    icon,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    color,
+    ...props
+  }: { icon: string; color?: Color } & React.ComponentProps<
+    typeof MaterialIcon
+  >) => <MaterialIcon {...props}>{icon}</MaterialIcon>,
+  {
+    shouldForwardProp: () => true,
```

Comment:
If this is just `true` think you can exclude it.

Reply:
I'll give it a go